cmd/admin-handlers.go

```
@@ -652,6 +671,20 @@ func (a adminAPIHandlers) HealHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if i := strings.Index(hip.clientToken, "@"); i >= 0 {
```

We should fail if the format is invalid with `invalidToken` error @vadmeste